benefits/static/css/styles.css

```diff
@@ -580,6 +608,12 @@ footer .footer-links li a:visited {
   box-shadow: inset 0 0 0 var(--border-width) var(--bs-white);
 }
 
+.radio-input:focus,
+.radio-input:focus-visible {
+  outline: 2px solid var(--focus-color) !important;
```

Comment:
I noticed the radio input focus rings are skinnier than everything else's focus rings. Should this be `3px`?

### What I see locally
- Radio button focus ring ![image](https://github.com/cal-itp/benefits/assets/25497886/53cca161-9bf8-4b62-83f4-d1fcc20cd18d)
- Compared to the link focus ring 
![image](https://github.com/cal-itp/benefits/assets/25497886/a517883e-9430-4bb8-b15b-717d324b4376)

### Set to 3px
![image](https://github.com/cal-itp/benefits/assets/25497886/7aac4841-4437-437d-a404-abfa0a5e5368)

### Figma
![image](https://github.com/cal-itp/benefits/assets/25497886/ea2cc30a-5928-46cb-a694-388311ed8ce5)

Reply:
It's hard to figure out the width of the yellow circle's stroke. But I got the 2px from this:
<img width="740" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/ee99c1d7-91e3-4eed-870f-561eaf961315">

```svg
<svg width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
  <circle cx="15" cy="15" r="8" fill="#323A45"/>
  <circle cx="15" cy="15" r="11" stroke="#323A45" stroke-width="2"/>
  <circle cx="15" cy="15" r="14" stroke="#FDB714" stroke-width="2"/>
</svg>
```

This line specifically:
`  <circle cx="15" cy="15" r="14" stroke="#FDB714" stroke-width="2"/>`

The yellow circle has a stroke-width of 2px, so that's why I set the radio button focus ring to 2px.

@srhhnry Should the focus ring for radio input be 2px (matches Figma) or 3px (matches all the other rings)?

Reply:
@machikoyasuda Hmm, yeah I see how the Figma says the radio input border should be 2px and all the others are 3px. I wonder why 2px is causing it to look so narrow on the radio input.

Reply:
> Should the focus ring for radio input be 2px (matches Figma) or 3px (matches all the other rings)?

@srhhnry Just to clarify this question, I don't think we're questioning the use of 2px in Figma for the radio input, but rather it seems like for some reason Figma and the browser are not showing 2px the same way. So I think what we're trying to decide is if we should just use 2px even though it doesn't look quite like Figma or use 3px even though that's not what Figma says.

Reply:
@machikoyasuda Maybe we merge this in so Sarah (or whoever on the design team 😄) can take a look at it on the dev site and decide if it looks too narrow?

Reply:
Yea - I'd also bet if we put our Mac vs Windows machines side by side, it might look a lil different between our machines. The input is gonna look different from Figma cos the Figma input was created from scratch. And your screenshots look a lil different from my Mac/Firefox rendering too.

Reply:
<img width="277" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/c2305a16-49f9-4f07-9092-749372b15f1b">
on Mac/Safari


<img width="257" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/aa3a33da-e230-429d-8eff-06fd9b40ea8b">
on Mac/Firefox

Reply:
Agreed. Ok, I feel good about leaving this as 2px and having design look at it later, either as a part of a design QA or whenever design has time.